### ISSUE TITLE
https instead of http

### ISSUE THREAD

**Issue by rhuss** [fstab/docker-weechat-web#2]
I would be supercool if https would be used so that OTR makes sense from the very beginning to the very end, especially if this image is used remotely.

**Comment by fstab** [comment]
Yes, I was also thinking about that. What I want to try is to set up an Nginx reverse proxy for that. The idea is that the docker container is only available locally, and does not do any authentication or encryption. The Nginx would be the bridge to the Internet and provide the security features. I will post an update when I tried it.
